src/pubsub/mod_pubsub_db.erl

```diff
@@ -20,23 +20,29 @@
 
 -callback stop() -> ok.
 
+%% ----------------------- Fun execution ------------------------
+
 -callback transaction(Fun :: fun(() -> {result | error, any()})) ->
     {result | error, any()}.
 
 %% Synchronous
 -callback dirty(Fun :: fun(() -> {result | error, any()})) ->
     {result | error, any()}.
 
--callback set_state(State :: mod_pubsub:pubsubState()) -> ok.
+%% ----------------------- Direct #pubsub_state access ------------------------
 
+%% TODO: Replace with del_node when it is fully possible from backend
+%%       i.e. when pubsub_item is migrated to RDBMS as well
 -callback del_state(Nidx :: mod_pubsub:nodeIdx(),
-                    UserLJID :: jid:ljid()) -> ok.
+                    LJID :: jid:ljid()) -> ok.
```

Comment:
If other callbacks expect `jid:jid` type, can we make this one consistent with the other and expect `jid:jid` as well?

Reply:
Nope, because it is called only in one place, where only `ljid()` is accessible directly so I'd have to convert it to `jid()`. This function will be removed anyway as soon as `pubsub_item` table is migrated to RDBMS as well.